docs/release_docs/Uat-ci-cd_ReleaseFLow.md

```diff
@@ -0,0 +1,33 @@
+## UAT-CI-CD
+
+There are two ways to use or trigger uat-CI-CD workflow
+1. By creating a Pull Request to merge into MAIN branch
+2. Whenever a Branch with prefix of 'release' create a pull request
```

Comment:
Whenever a pull request is created with source branch having the `release/` prefix. 

NOTE: Remember, in our case, the prefix we are following is `release/` and not `release`. If you say `release` then anyone with a branch `release-mohit` should be able to run it which is not allowed